app/views/upcoming_offline_reservation_mailer/send_offline_instrument_warning.html.haml

```diff
@@ -0,0 +1,6 @@
+%p
+  = html "upcoming_offline_reservation_mailer.send_offline_instrument_warning.body.html",
```

Comment:
Can we use the shorthand? That would mean putting this under `views.upcoming_offline_reservation_mailer.send_offline_instrument_warning`.